docs/source/basic-info/about.rst

```diff
@@ -0,0 +1,60 @@
+########
```

Comment:
I think this is the old documentation page.  Want to be sure that this doesn't clobber the updated doc.